content/en/getting_started/cloud_security_management/_index.md

```diff
@@ -36,11 +36,11 @@ This guide walks you through best practices for getting your team up and running
 ## Phase 1: Deployment
 
 1. Install the [Datadog Agent (version 7.46 or above)][4].
-2. [Enable CSM for for your cloud resources and infrastructure][5]:
+2. [Enable CSM for your cloud resources and infrastructure][5]:
     - **CSM Threats**: Kubernetes, Docker, and host-based installations.
     - **CSM Misconfigurations**: AWS, Azure, GCP, Kubernetes, and Docker instructions.
     - **CSM Identity Risks**: Enable AWS resource collection and Cloudtrail logs forwarding.
-    - **CSM Vulnerabilities**: Container image scanning and host scanning instructions for Kubernetes, ECS EC2 instances, and host-based installations.
+    - **CSM Vulnerabilities**: Container image scanning and host scanning instructions for Kubernetes, ECS EC2 instances, host-based installations and AWS (agentless).
```

Comment:
Should we add a link on agentless leading to https://www.datadoghq.com/blog/agentless-scanning/ ?

Reply:
Blog links typically go in the `Further reading` section. I've opened a PR to add that here:
https://github.com/DataDog/documentation/pull/24956